hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1076,6 +1077,12 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{RequeueAfter: duration}, nil
 	}
 
+	if _, exists := hcluster.Annotations[ETCDRestoreFailingMemberAnnotation]; exists {
```

Comment:
should we use the annotation value to signal the intention / state? 

e.g.:
```
"hypershift.openshift.io/etcd-restore-failing-member" : "true"  # SRE sets this to indicate the desire for the automation to kick in.
"hypershift.openshift.io/etcd-restore-failing-member" : "error"  # Set by the HO to indicate to SRE that this etcd recovery failed
"hypershift.openshift.io/etcd-restore-failing-member" : "done_20240803160405"  # Set by the HO to indicate that it succeeded at the shown timestamp (YYYYMMDDhhmmss)
```

I think this makes this operation more observable and traceable.